tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs

```diff
@@ -50,8 +51,7 @@ public void ParseEpisodesCorrectly(string path, bool isDirectory, string name, i
         [InlineData("/test/01-03.avi", true, true)]
         public void EpisodePathParserTest_DifferentExpressionsParameters(string path, bool? isNamed, bool? isOptimistic)
         {
-            NamingOptions o = new NamingOptions();
-            EpisodePathParser p = new EpisodePathParser(o);
+            EpisodePathParser p = new EpisodePathParser(_namingOptions);
```

Comment:
if we add a constructor and dispose, we can initialize the EpisodePathParser in the constructor and remove all the repetitive lines.
private readonly EpisodePathParser _episodePathParser;
`
public EpisodePathParserTest() {
 _episodePathParser = new EpisodeResolver(_namingOptions);
}`

But I'm not sure if that will improve the speed

Reply:
`NamingOptions` is very expensive to create, this won't make much difference